utils/pipeline_status_update.py

```diff
@@ -94,9 +94,9 @@ def fetch_file_content():
     else:
            version_name = "Release Version"
     # updating result's table if the table is already present
-    if file_content.find('<table>\n <tr>\n  <th>Pipeline ID</th>\n  <th>Execution Time</th>\n  <th>'+version_name+'</th></tr>\n')>0:
-        new_pipeline = ' <tr>\n  <td>{}</td>\n  <td>{}</td>\n  <td>{}</td>\n </tr>\n'.format(pipeline_url,time_stamp,tag)
-        index = content_list.index('  <th>'+version_name+'</th></tr>')
+    if file_content.find('<table>\n <tr>\n  <th>Pipeline ID</th>\n  <th>Execution Time</th>\n  <th>'+version_name+'</th>\n <th>Coverage</th>\n </tr>\n')>0:
+        new_pipeline = ' <tr>\n  <td>{}</td>\n  <td>{}</td>\n  <td>{}</td>\n<td>{}</td> </tr>\n'.format(pipeline_url,time_stamp,tag,totalCoverage)
```

Comment:
```suggestion
        new_pipeline = ' <tr>\n  <td>{}</td>\n  <td>{}</td>\n  <td>{}</td>\n<td>{}</td>\n </tr>\n'.format(pipeline_url,time_stamp,tag,totalCoverage)
```
We might need one new line also.